docs/how-to/enforce-permissions/authorized-users.mdx

```diff
@@ -119,6 +119,29 @@ the response will also contain the tenant level role assignment, and will look a
 }
 ```
 
-:::info
-Currently, using "authorized_users" works only for RBAC & ReBAC.
-:::
+## ABAC Usage
```

Comment:
add some more details here, mention that this is flagged and the performance overhead of it with some 
```
:::warning
:::
```